docs/report.md

```diff
@@ -72,8 +72,13 @@ State which software license you chose for your application.
 
 ### LLMs, ChatGPT, CoPilot, and others
 
-State which LLM(s) were used during development of your project.
-In case you were not using any, just state so.
-In case you were using an LLM to support your development, briefly describe when and how it was applied.
-Reflect in writing to which degree the responses of the LLM were helpful.
-Discuss briefly if application of LLMs sped up your development or if the contrary was the case.
+In the development process LLM were used sparingly to support the coding process.
+GitHub copilot and the JetBrains AI was occasionally used to finish lines of code, when it came with good suggestions. 
```

Comment:
We havent used Jetbrains AI in the development of Chirp, that is a paid feature that most of the group does not have access to, and the one person that does, says they havnt used it.